src/routes/feed.xml/+server.ts

```diff
@@ -0,0 +1,50 @@
+import RSS from 'rss';
+import type { BlogPost } from '$lib/public/blog/types';
+import { discernLabels } from '$lib/public/blog/utils';
+import { fetchBlogPosts } from '$lib/server/blog/posts';
+
+export async function GET() {
+  // TODO: Mix events and blog posts.
```

Comment:
Consider: In `/feed.xml`, should we mix event entries with blog post entries in the RSS?

Reply:
This change will occur in a future PR after further discussion.